packages/nextra/src/plugin.ts

```diff
@@ -86,36 +98,75 @@ export async function collectFiles(
         fileMap[fp] = {
           kind: 'Meta',
           ...(locale && { locale }),
-          data: parseJsonFile(content, fp)
+          data: JSON.parse(content)
```

Comment:
here `parseJsonFile` is used to avoid throwing an error in case of incorrect JSON file